source.csv

```diff
@@ -134,3 +134,4 @@ Yana Welinder,Kraftful,CEO / Cofounder,yanatweets,yanawelinder
 Yann LeCun,Meta/NYU,Chief AI Scientist/Professor,ylecun,yannlecun
 Yoshitomo Matsubara,Amazon,Alexa AI Applied Scientist,yoshitomo_cs,yoshitomo_cs
 Zubair Irshad,Georgia Tech,Robo/ML PhD Student,mzubairirshad,imzubairai
+Michael Gold,School of Visual Arts / Shaderverse,Professor / Cofounder,michaelgold,michaelgold
```

Comment:
It looks like `michaelgold` is not an account on threads. did you mean to add an underscore at the end?

```suggestion
Michael Gold,School of Visual Arts / Shaderverse,Professor / Cofounder,michaelgold,michaelgold_
```

Reply:
Hey there. Maybe they are having server issues. My account is `michaelgold`. Here is the link to my threads account. https://www.threads.net/@michaelgold

Reply:
looks like server issues. I found you.